telemetry/definitions/commonDefinitions.json

```diff
@@ -850,6 +880,29 @@
             "name": "apprunner_viewServiceLogs",
             "description": "View the App Runner service logs (the logs produced by App Runner)"
         },
+        {
+            "name": "auth_addConnection",
+            "description": "Covers the user's 'Add New Connection' experience from an entry point. Captures a snapshot of user state on completion.",
+            "metadata": [
+                { "type": "source"},
+                { "type": "authUiElement", "required": false},
+                { "type": "unlockedTabs", "required": false},
+                { "type": "authConnectionsCount", "required": false},
+                { "type": "credentialSourceId", "required": false},
+                { "type": "invalidInputFields", "required": false},
+                { "type": "awsRegion", "required": false },
```

Comment:
`awsRegion` is global so it can be added to every event. I'd remove this for now because it breaks some of the language-specific tests:
```suggestion
```